app/client/components/helpCentre/HelpTopicBox.tsx

```diff
@@ -0,0 +1,150 @@
+import { css } from "@emotion/core";
+import { palette, space } from "@guardian/src-foundations";
+import { textSans } from "@guardian/src-foundations/typography";
+import Color from "color";
+import React from "react";
+import { minWidth } from "../../styles/breakpoints";
+import { trackEvent } from "../analytics";
+import { getHelpSectionIcon } from "../svgs/helpSectionIcons";
+import { HelpCentreTopic } from "./helpCentreConfig";
+
+interface HelpTopicBoxProps {
+  topic: HelpCentreTopic;
+}
+
+export const HelpTopicBox = (props: HelpTopicBoxProps) => {
+  return (
+    <div
+      css={css`
+        border: 1px solid ${palette.neutral["86"]};
```

Comment:
Can we move these big chunks of CSS into their own variables to make it easier to read what this component returns?